README.md

```diff
@@ -1,39 +1,64 @@
-# Emacs major mode for the julia programming language
+# julia-mode
+
+
+<div align="center"><img src="logo.svg" height="150" alt="julia-emacs julia-mode logo"/></div>
+
+<br />
```

Comment:
I would suggest aligned left (default), and simple Markdown:
```suggestion
![julia-emacs julia-mode logo](logo.svg)
```

Reply:
Changed as suggested. I also replaced the svg with a png, as Github Markdown doesn't do resizing and changing native size of svg didn't seem to have an effect.